public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -97,6 +104,47 @@ export function LogsTable(props: Props) {
     [logsSortOrder, timeZone, splitOpen, range, logsFrame?.bodyField.name, logsFrame?.timeField.name, timeIndex]
   );
 
+  useEffect(() => {
+    if (logsDrilldownData) {
+      console.log('LogsDrilldown API Response:', logsDrilldownData);
+    }
+    if (logsDrilldownError) {
+      console.error('LogsDrilldown API Error:', logsDrilldownError);
+    }
+  }, [logsDrilldownData, logsDrilldownError]);
+
+  // Create LogsDrilldown with default fields if none exist
+  useEffect(() => {
+    const createDefaultLogsDrilldown = async () => {
+      // Check if items array is empty (no resources exist)
+      if (logsDrilldownData && logsDrilldownData.items && logsDrilldownData.items.length === 0) {
+        try {
+          const namespace = getAPINamespace();
+          const result = await createLogsDrilldown({
+            logsDrilldown: {
+              apiVersion: 'logsdrilldown.grafana.app/v1alpha1',
+              kind: 'LogsDrilldown',
+              metadata: {
+                namespace,
+                generateName: 'logs-drilldown-',
+              },
+              spec: {
+                defaultFields: ['time', 'body', 'level'],
+              },
```

Comment:
I know this is just a test, but if this is how you see this being integrated, these are not defaultFields but displayedFields or selectedColumns.